test/nice_partials/partial_test.rb

```diff
@@ -164,6 +164,17 @@ def call
     assert_equal "Hello there", partial.title.to_s
   end
 
+  test "helpers can access local_assigns" do
+    partial = new_partial(locals: {foo: "bar"})
+    partial.helpers do
```

Comment:
I'm wondering if this will work in an app since we automatically yield before the helpers block would've been executed? Maybe we should convert this to a renderer_test based test to berify that? Have you managed to see your changelog code sample work in your app?

Reply:
I've added coverage to the renderer test to exercise this scenario. Does it look realistic enough?